codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionTest.java

```diff
@@ -899,4 +909,22 @@ private void verifyParentChanging(Http2Stream stream, Http2Stream newParent) {
     private void verifyParentChanged(Http2Stream stream, Http2Stream oldParent) {
         verify(clientListener).onPriorityTreeParentChanged(streamEq(stream), streamEq(oldParent));
     }
+    private Http2Stream child(Http2Stream parent, final int id) {
+        try {
+            final Holder<Http2Stream> streamHolder = new Holder<Http2Stream>();
+            parent.forEachChild(new Http2StreamVisitor() {
+                @Override
+                public boolean visit(Http2Stream stream) throws Http2Exception {
+                    if (stream.id() == id) {
+                        streamHolder.value = stream;
+                        return false;
+                    }
+                    return true;
+                }
+            });
+            return streamHolder.value;
+        } catch (Http2Exception e) {
+            throw new RuntimeException(e);
```

Comment:
This is actually uglier code since you then have to `return null`, which will never be executed.  Ideally the `throwException(e)` method should return an exception so the code would be:

`throw PlatformDependent.throwException(e)`. 

Anyway, I've updated the code to return null.